protos/feast/core/Store.proto

```diff
@@ -118,8 +126,15 @@ message Store {
   }
 
   message CassandraConfig {
-    string host = 1;
+    // - bootstrapHosts: [comma delimited value of hosts]
+    string bootstrap_hosts = 1;
     int32 port = 2;
+    string keyspace = 3;
+    string table_name = 4;
+    map<string, string> replication_options = 5;
+    // Default expiration in seconds to use when FeatureSetSpec does not have max_age defined.
+    // Specify 0 for no expiration
+    google.protobuf.Duration default_ttl = 6;
```

Comment:
@zhilingc do you think it makes sense to document the use of max age for TTL in Cassandra here? Is it visible enough? 
The comments for this already hints the use of max age but I can make it more clear.

Reply:
Can we also document `replication options` and `table_name`fields?

For instance by providing a reference to show what are the valid key and values in the `replication_options`:
https://docs.datastax.com/en/dse/6.7/cql/cql/cql_reference/cql_commands/cqlCreateKeyspace.html#cqlCreateKeyspace__cqlCreateKeyspacereplicationmap-Pr3yUQ7t

Also for `table_name` as for current implementation, only **feature_store** as table name is valid.

Reply:
Sure, I had put this into `CONTRIBUTION.md` but best to put in all together here. I'll update this too.

Reply:
Ok thanks